glustercli/cmd/volume-set.go

```diff
@@ -31,6 +31,7 @@ func init() {
 	volumeSetCmd.Flags().BoolVar(&flagSetAdv, "advanced", false, "Allow setting advanced options")
 	volumeSetCmd.Flags().BoolVar(&flagSetExp, "experimental", false, "Allow setting experimental options")
 	volumeSetCmd.Flags().BoolVar(&flagSetDep, "deprecated", false, "Allow setting deprecated options")
+	volumeSetCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "verbose output")
```

Comment:
This is done at a lot of places. Cobra has provision to set a flag on the root command and all subcommands will inherit that flag.

See `glustercli/cmd/root.go`:

```go
RootCmd.PersistentFlags().BoolVarP(...
```